Fix slashes test (#347)
encukou/freezeyt: README.md

(For pages that are part of your website, we recommend
adding them to your application rather than as extra files.)

> If you specify `url part` of `extra file` with two or more right-slashes the `kanrisha`
> will reduce to one.
> 
> If you specify left-slashes in `url part` the `kanrisha` convert it to right-slashes (POSIX variant).
> 

Reviewer: ```suggestion
If you specify backslashes in `url part`, `freezeyt` convert them to forward slashes.

```
